runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -632,6 +616,7 @@ double __quantum__qis__Variance(ObsIdType obsKey)
 void __quantum__qis__Probs(MemRefT_double_1d *result, int64_t numQubits, ...)
 {
     assert(numQubits >= 0);
+    MemRefT<double, 1> *result_p = (MemRefT<double, 1> *)result;
```

Comment:
Why are you casting this, isn't already the same type?

Note the `MemRefT_double_1d` and similar types are used to satisfy the C-only public API.

Reply:
Thanks for the explanation, makes sense :)